channels/management/commands/runserver.py

```diff
@@ -131,39 +131,34 @@ def log_action(self, protocol, action, details):
         """
         Logs various different kinds of requests to the console.
         """
-        # All start with timestamp
-        msg = "[%s] " % datetime.datetime.now().strftime("%Y/%m/%d %H:%M:%S")
         # HTTP requests
         if protocol == "http" and action == "complete":
-            msg += (
-                "HTTP %(method)s %(path)s %(status)s [%(time_taken).2f, %(client)s]\n"
-                % details
-            )
+            msg = "HTTP %(method)s %(path)s %(status)s [%(time_taken).2f, %(client)s]"
+
             # Utilize terminal colors, if available
             if 200 <= details["status"] < 300:
                 # Put 2XX first, since it should be the common case
-                msg = self.style.HTTP_SUCCESS(msg)
+                logger.info(msg, details)
```

Comment:
You've lost the colour information - Django runserver still has it last time I checked, so we need to match that.

Reply:
Yeah I missed that. Fixing in next commit